README.rst

```diff
@@ -23,20 +23,31 @@ Building is as simple as
 ::
 
         $ git clone https://github.com/FreeRCT/FreeRCT.git
-        $ cd FreeRCT # Go into the downloaded source directory.
-        $ cmake .    # Checks libraries are where they're supposed to be and replaces some strings
-        $ make       # Let make do the heavy work.
+        $ cd FreeRCT    # Go into the downloaded source directory.
+        $ cmake .       # Checks libraries are where they're supposed to be and replaces some strings.
+        $ make          # Let make do the heavy work.
+        $ make install  # Optional step to install the program on the system. May require root privileges.
 
 
 CMake accepts the following options:
 
-================= ======= ==========================================================================
-Name              Default Explanation
-================= ======= ==========================================================================
-OVERRIDE_VCS      OFF     Override the lack of a VCS checkout.
-ASAN              OFF     Use AddressSanitizer (https://clang.llvm.org/docs/AddressSanitizer.html).
-USERDATA_PREFIX   '.'     Directory where user data such as savegames is stored.
-================= ======= ==========================================================================
+======================= ============================= ================================================================================
+Name                    Default value                 Explanation
+======================= ============================= ================================================================================
+OVERRIDE_VCS            OFF                           Override the lack of a VCS checkout. Supported VCSs are git and bzr.
+VERSION_STRING          '0.0.0-norev'                 If OVERRIDE_VCS is enabled, use this string as the build version.
+ASAN                    OFF                           Use AddressSanitizer (https://clang.llvm.org/docs/AddressSanitizer.html).
```

Comment:
As I didn't have any idea what ASAN is, this line reads like: `'X': "use X (see http://.../X)"`. This line doesn't explain what X does, or why X is useful, or who is supposed to use this X thing. From only "X: use X", there is no clue for deciding whether I should make the effort of adding this option.  
You provided a full name and link and that's great, but "address" is a quite broad topic, and combined with 'sanitize' I went towards normalizing URLs or email addresses, which doesn't make sense to me at all.  
I can open the web page, but it would be very useful to at least know beforehand what the topic is going to be.

Note that the above is a common trap for people writing documentation,  you see it everywhere if you know it exists. The trap is the implicit assumption that all readers have the same knowledge as the author. Obviously, that cannot be true (fortunately).

----
There are 2 ways out of this. One option is to explain somewhere what ASAN is in a few words. That gives readers a fighting chance of deciding whether ASAN is of use to them.

Another option is to generalize a bit. You don't want `ASAN`, you want a `MEMORYCHECKER` (or `MEMCHECK` or so), and today we support `ASAN`, ie `MEMORYCHECK=ASAN`. That gives much more information to a reader. Even then, likely the goal of 'MEMORYCHECK' needs to be explained as there is room for different interpretations as well. (Checking whether the memory itself is sane versus sane behavior of the program towards memory use.)

Reply:
Updated the Readme with a section about ASan. It's a very useful development tool so a paragraph on it can't hurt.